internal/command/command.go

```diff
@@ -28,7 +29,17 @@ import (
 // Run runs generic test.
 // It runs a command with arguments in a directory and returns the combined output as is.
 // If the command exits with a non-zero exit code, the test fails.
+// If all args are executable files with .sh extension it runs each file in a separate CommandContext.
 func Run(ctx context.Context, dir string, args []string) (*config.TestResults, error) {
+	allExecutableFiles := !slices.ContainsFunc(args, func(arg string) bool {
```

Comment:
No, we are not doing that. `command` handler should run one command. We are not running one or multiple commands based on command arguments.

Please create separate YAML configurations or find some other solution.